DESCRIPTION

```diff
@@ -17,3 +17,7 @@ License: MIT + file LICENSE
 URL: https://r6.r-lib.org, https://github.com/r-lib/R6/
 BugReports: https://github.com/r-lib/R6/issues
 RoxygenNote: 7.1.2
+Config/Needs/website:
```

Comment:
I think lobstr can also be moved here, since it's only used for the vignettes (and the vignettes are only used for the website, I believe, and not as actual vignettes).

Reply:
Great!

Reply:
Ah, `lobstr` is used in a test. Leaving where it is.